twinkle.js

```diff
@@ -397,7 +397,7 @@ $.ajax({
 	dataType: 'text'
 })
 	.fail(function () {
-		mw.notify('Could not load your Twinkle preferences', {type: 'error'});
+		mw.notify('Could not load your Twinkle preferences');
```

Comment:
That makes sense, I suppose, but given that it'll be working *differently* than folks might suspect, I think it should at least include a note to that effect, e.g.
```suggestion
		mw.notify('Could not load your Twinkle preferences, resorting to default Twinkle preferences');
```
Is it worth noting that a reload might (likely?) help?  We could also extend the length it appears, or even require clicking to dismiss; not sure those are necessary.

Reply:
Done. Requiring click to dismiss would be quite annoying on mobile (where this seems to occur more often).